denonavr/denonavr.py

```diff
@@ -43,11 +43,14 @@
      ('GAME', ['PLII GAME', 'DOLBY D +NEO:X G', 'VIDEO GAME']),
      ('AUTO', ['None']),
      ('VIRTUAL', ['VIRTUAL']),
-     ('PURE DIRECT', ['DIRECT']),
+     ('PURE DIRECT', ['DIRECT', 'PURE_DIRECT']),
      ('DOLBY DIGITAL', ['DOLBY DIGITAL', 'DOLBY D + DOLBY SURROUND',
                         'DOLBY DIGITAL +', 'DTS NEURAL:X', 'STANDARD(DOLBY)',
-                        'DOLBY SURROUND']),
-     ('MCH STEREO', ['MULTI CH STEREO', 'MULTI CH IN']),
+                        'DOLBY SURROUND', 'STANDARD(DTS)', 'DTS SURROUND',
+                        'DTS + NEURAL:X', 'DOLBY D + +DOLBY SURROUND',
```

Comment:
Out of curiosity why are the DTS soundmodes being grouped with the Dolby ones rather than keeping them separately?

Reply:
Because I am not sure if the "DOLBY DIGITAL" command can really be seperated with the "DTS SURROUND" command.
This documentation suggests that their could actually be a seperation: m.us.marantz.com/DocumentMaster/US/2013%20Marantz_AV_SR_NR_PROTOCOL_V07.xls

If you can test and show that the receiver does respond to a DTS SURROUND command I am happy to seperate the two.
At the moment I do not have time to investigate this on my receiver.

Reply:
@rohankapoorcom I just checked and indeed the "DTS SURROUND" command worked.
Thanks for pointing this out to me!
I just made a commit to split up the "Dolby Digital" and "DTS Surround" modes.